src/xenia/base/vec128.h

```diff
@@ -266,8 +266,8 @@ static inline vec128_t vec128b(uint8_t x0, uint8_t x1, uint8_t x2, uint8_t x3,
 
 inline std::string to_string(const vec128_t& value) {
   char buffer[128];
-  std::snprintf(buffer, sizeof(buffer), "(%g, %g, %g, %g)", value.x, value.y,
-                value.z, value.w);
+  snprintf(buffer, sizeof(buffer), "(%g, %g, %g, %g)", value.x, value.y,
```

Comment:
I don't think this belongs to this PR. Why did you drop `std::`?

Reply:
Clarified the drop of std:: in the commit message, copying here:
This seems to be an issue with Microsoft declaring a snprintf macro in an included header which causes issues.
Related issues:

nlohmann/json#1408
kodi-pvr/pvr.vuplus#162